dlt/common/schema/utils.py

```diff
@@ -317,6 +318,19 @@ def validate_stored_schema(stored_schema: TStoredSchema) -> None:
             if parent_table_name not in stored_schema["tables"]:
                 raise ParentTableNotFoundException(table_name, parent_table_name)
 
+        # check for "replicate" tables that miss a primary key or "cdc_config"
```

Comment:
this makes sense but we should move it to
1. end of normalize stage OR
2. beginning of load stage
at this moment schema may be still partial. not all columns may be present (100% after extract stage)

also we should check merge disposition.

also take a look at `_verify_schema` in `JobClientBase` looks like our place :)